dashboard/admin.py

```diff
@@ -54,15 +66,18 @@ class CourseAdmin(admin.ModelAdmin):
     readonly_fields = ('term',)
 
     # Need this method to correctly display the line breaks
-    def _courseviewoption(self, obj):
+    @staticmethod
+    @admin_attr_decorator
+    def _courseviewoption(obj: Course) -> mark_safe:
         return mark_safe(linebreaksbr(obj.courseviewoption))
     _courseviewoption.short_description = "Course View Option(s)"
 
-    def course_link(self, obj):
+    @staticmethod
+    def course_link(obj: Course) -> format_html:
```

Comment:
When I log the type of the returned values, I see `<class 'django.utils.safestring.SafeText'>`.

Reply:
Thanks, I'll update this